### ISSUE TITLE
futures-await-async-macro 0.1.3 no longer accepts #[async(boxed)]

### ISSUE THREAD

**Issue by estokes** [alexcrichton/futures-await#103]
This is on nightly 2018-04-11, futures-await itself does not appear to compile on a current nightly. I get that there is going to be churn while the grand async plan is implemented, but anything that could be done to minimize that would be greatly appreciated! Some of us have code that depends on the existing libraries after all.

```rust
error: custom attribute panicked
  --> libgsh/src/resolver_client.rs:51:3
   |
51 |   #[async(boxed)]
   |   ^^^^^^^^^^^^^^^
   |
   = help: message: the #[async] attribute currently only takes `boxed` as an arg

error: aborting due to previous error

error: Could not compile `libgsh`.
```
As a work around one can just force cargo to use 0.1.2.

**Comment by withoutboats** [comment]
See #102, this is churn in the proc macro API.

**Comment by estokes** [comment]
Ok understood, I can just work around it for now.

**Comment by withoutboats** [comment]
`futures-await-async-macro` has been fixed in 0.1.4, update your dependency with `cargo update -p futures-await-async-macro` :)
